src/qibocal/auto/transpile.py

```diff
@@ -4,17 +4,57 @@
 from qibo.backends.abstract import Backend
 from qibo.transpiler.pipeline import Passes
 from qibo.transpiler.unroller import NativeGates, Unroller
+from qibolab.qubits import QubitId
+
+
+def transpile_circuits(
+    circuits: list[Circuit],
+    qubit_maps: list[list[QubitId]],
+    backend: Backend,
+    transpiler: Optional[Passes],
+):
+    """Transpile and pad `circuits` according to the platform.
+
+    Apply the `transpiler` to `circuits` and pad them in
+    circuits with the same number of qubits in the platform.
+    Before manipulating the circuits, this function check that the
+    `qubit_maps` contain string ids and in the positive case it
+    remap them in integers, following the ids order provided by the
+    platform.
+
+    .. note::
+
+        In this function we are implicitly assume that the qubit ids
+        are all string or all integers.
+    """
+    transpiled_circuits = []
+
+    qubits = list(backend.platform.qubits)
+    if isinstance(qubit_maps[0][0], str):
+        for i, qubit_map in enumerate(qubit_maps):
+            qubit_map = map(lambda x: qubits.index(x), qubit_map)
+            qubit_maps[i] = list(qubit_map)
+    if backend.name == "qibolab":
```

Comment:
Instead, just early return
```suggestion
    if backend.name != "qibolab":
        return circuits
```

Reply:
I thought that it was fine but this will case CI to fail https://github.com/qiboteam/qibocal/actions/runs/10106037732/job/27947515680.
I'm not sure why.